### PR TITLE
Updated info;tips > to include information on how to add an additional disk to ITS.

### PR DESCRIPTION
**Pull request by eswenson1**

### CODE REVIEW

File: doc/info/tips.2 [hunk 1]
```diff
@@ -239,3 +242,225 @@ You're now in DDT and you can examine ITS, set breakpoints, etc. Type $g to star
...
+   y
+   ^C
+
+3. Wait until ITS has shut down.
+
+

+File: TIPS,  Node: Update emulator configuration for new disks, Previous: Shut down ITS, Up: Adding an additional disk, Next: Boot SALV or NSALV
+
+This procedure is differnt for the various emulators used to boot ITS. For KLH10, update the dskdmp.ini file to
```
Comment (larsbrinkhoff) [inline doc/info/tips.2:325]: differnt -> different

**Review by larsbrinkhoff — APPROVED** [review]
Great level of detail!  @bictorv, does this work for you?

**Comment by bictorv** [comment]
Thanks! I'll have a shot at it in the coming days, and see what I discover while doing it.

**Review by bictorv — APPROVED** [review]

File: doc/info/tips.2 [hunk 2]
```diff
@@ -239,3 +242,225 @@ You're now in DDT and you can examine ITS, set breakpoints, etc. Type $g to star
 6.  When the new ITS has passed testing, rename the old .; @ ITS to .; @ OITS.  Rename the
 new ITS to .; @ ITS.
 
+
+File: TIPS,  Node: Adding an additional disk, Next: Build new version of SALV or NSALV, Up: TOP
+
+* Menu:
+
+* Build new version of SALV or NSALV
```
Comment (bictorv) [inline doc/info/tips.2:250]: These should have an ending double colon (::) to be parsable by Info, I think.

**Pull request merged**